### ISSUE TITLE
Issue deriving sd card key

### ISSUE THREAD

**Issue by pcartwright81** [Thealexbarney/LibHac#237]
If I use the dumped prod keys with the sd key derive keys fails.  If I pass in the sd seed from prod keys it works.  Debugged into libhac, but I do not know enough about the keys to understand why this is happening, or why the sd seed is not read from prod.keys.

**Comment by Thealexbarney** [comment]
Does the prod.keys from lockpick contains the correct sd card seed? The same one you use when you pass it manually?

**Comment by pcartwright81** [comment]
Yes, and it is the same key that is in  ref _keys.DeviceKeys.SdCardEncryptionSeed after the keys are derived.  I dumped with Lockpick_RCM 1.9.6 and I am on Firmware > 13.x.x.  I also tried copying the lines from prod.keys and creating a console.keys.
Errors I get in the console are:
Failed to match key device_key_4x
Failed to match key eticket_rsa_kek_source
Failed to match key eticket_rsa_kekek_source
Failed to match key rsa_oaep_kek_generation_source
Failed to match key rsa_private_kek_generation_source
Failed to match key save_mac_key
Failed to match key ssl_rsa_kek_source_x

For this dump the prod keys have all of the console specific information.

**Comment by Thealexbarney** [comment]
Did you ever figure out what was causing this issue?

**Comment by pcartwright81** [comment]
I started using another way to get the NSP off of the SD card, I don't think I have tried using this method again.
